runtime/runtime-standalone/src/lib.rs

```diff
@@ -16,9 +16,11 @@ use near_primitives::types::{
     StateChangeCause,
 };
 use near_primitives::version::PROTOCOL_VERSION;
+use near_primitives::views::ViewApplyState;
```

Comment:
This crate was moved to the sdk.  I'll update my PR to use this after it merges.

Reply:
should we delete runtime-standalone from nearcore to avoid keep maintain two copy of it?